docs/tutorial-modifier-roles/add-role.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 3
+---
+
+# Add Role
+
+After confirming the transaction, you should return to the Zodiac App, where your configured Role Modifier will be displayed in the left side panel. Click anywhere on the modifier's card to access the Roles App.
+
+![Roles App - zero state](/img/tutorial/roles_03.png)
+
+## Create a Role
+
+![Roles App - zero state](/img/tutorial/roles_04.png)
+Click on the "Create a Role" button to open the Role view where you can add members and targets to the role. The Role name is created automatically.
```

Comment:
@cedricwaxwing I don't know if we should refer to a "name" since in reality, we use an index... 🤔

Reply:
Maybe this is better? "The Role name is created automatically by using the role's index." I feel like the user will still likely think of this as the "Role name" even if it is literally just an index.